### ISSUE TITLE
Create skiplink to main content for keyboard navigation

### ISSUE THREAD

**Issue by jonels-msft** [readthedocs/sphinx_rtd_theme#972]
### Problem

Screen reader users are inconvenienced, as they have to navigate repeated menu content on every page.

### Reproducible Project

https://pg-auto-failover.readthedocs.io/en/latest/

#### Error Logs/Results

![skip-to-main](https://user-images.githubusercontent.com/48225118/86811306-5f27b100-c043-11ea-97f9-d19c2238fc25.png)

#### Expected Results

[WCAG 2.4.1](https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks.html) explains how to enable screen reader users to bypass blocks and skip to content.

It looks like there was some progress for this in https://github.com/readthedocs/sphinx_rtd_theme/pull/730 but that PR was abandoned.

### Environment Info

- Python Version: 3.7.3
- Sphinx Version: 1.8.5
- RTD Theme Version: 0.4.3
